test/test_helper.rb

```diff
@@ -1,39 +1,36 @@
 # frozen_string_literal: true
 
-require "bundler/setup"
-
-Bundler.require(:default)
-
+$LOAD_PATH.unshift File.expand_path("../lib", __dir__)
+require "ruby_ui"
 require "phlex"
-# require "phlex/testing/view_helper"
+require "json"
+
 require "minitest/autorun"
 
-class TestContext < Phlex::HTML
-  def view_template(&)
-    div(&)
+module RubyUI
```

Comment:
This is needed to make `RubyUI.ComponentName` work. On user apps, we will generate something like:
```ruby
Rails.autoloaders.main.push_dir(
  "#{Rails.root}/app/components/ruby_ui", namespace: RubyUI
)
```